src/bio/bio_xstream.c

```diff
@@ -394,6 +394,12 @@ bio_nvme_init(const char *storage_path, const char *nvme_conf, int shm_id,
 		goto free_mutex;
 	}
 
+	if (strlen(nvme_conf) == 0) {
+		D_INFO("NVMe config isn't specified, skip NVMe setup.\n");
+		nvme_glb.bd_nvme_conf = NULL;
+		return 0;
+	}
+
 	fd = open(nvme_conf, O_RDONLY, 0600);
 	if (fd < 0) {
 		D_WARN("Open %s failed("DF_RC"), skip DAOS NVMe setup.\n",
```

Comment:
https://wiki.hpdd.intel.com/display/DC/C+Style+Guide says rc should be printed at the end.

```suggestion
		D_WARN("Open %s failed, skip DAOS NVMe setup "DF_RC"\n",
```